rest_api/limsdb/queries/sample_status.py

```diff
@@ -135,7 +135,7 @@ def started_date(self):
         """Date of the first completed step"""
         for p in reversed(self.processes):
             process, date, process_type, process_id = p
-            if process_type == 'complete':
+            if process_type == 'complete' and process_type in status_cfg.started_steps():
```

Comment:
Method call on a property?

Reply:
Oops sorry, thanks for pointing that out!